tests/util/checker/parser/flaky_test_finder.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2018 Istio Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package flakytestfinder
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"istio.io/istio/tests/util/checker"
+)
+
+// ReportFlakyTests reports names of tests that have annotation.IsFlaky() call.
+// nolint:deadcode
+func ReportFlakyTests() {
+	flag.Parse()
```

Comment:
since this is not main(), should it take input arguments instead of taking flag values? or remove this completely?

Reply:
This is removed and renamed getReport() to ReportFlakyTests().